specs/deposits.md

```diff
@@ -296,6 +296,16 @@ Total calldata length MUST be exactly 164 bytes, implying the sixth and final se
 partially filled. This helps to slow database growth as every L2 block includes a L1 Attributes
 deposit transaction.
 
+In the first L2 block after the Ecotone activation block, the Ecotone L1 attributes are first used.
+
+The pre-Ecotone values are migrated over 1:1. The new Ecotone attributes are set as following:
```

Comment:
Rather than talk about it as the "first" moment, could we say something like

> Blocks *after* the Ecotone activation block contain all pre-Ecotone values 1:1, and also set the following new attributes: [...]

I could be wrong here since your phrasing "are first used" indicate that the values might already be populated (just not used), but I am not sure.